lib/matplotlib/axes/_axes.py

```diff
@@ -2128,11 +2121,13 @@ def make_iterable(x):
                 edgecolor=e,
                 linewidth=lw,
                 label='_nolegend_',
-                margins=margins
                 )
             r.update(kwargs)
             r.get_path()._interpolation_steps = 100
-            #print r.get_label(), label, 'label' in kwargs
+            x_stickies, y_stickies = r.get_stickies()
+            (getattr(r.get_stickies(),
+                     {'vertical': 'y', 'horizontal': 'x'}[orientation])
+             .append(0))
```

Comment:
It looks like you have a 4-physical-line obfuscation of the following 5 lines, correct?
```python
x_stickies, y_stickies = r.get_stickies()
if orientation == 'vertical':
    y_stickies.append(0)
else:
    x_stickies.append(0)
```
But you only need x or y, so you could also do
```python
if orientation == `vertical`:
    r.get_stickies().y.append(0)
else:
    r.get_stickies().x.append(0)
```
so we are back down to 4 physical lines that I find much easier to understand.  And it would look even nicer with `stickies` as a property.